index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <title>Fork Me! FCC: Test Suite Template</title>
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+ <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+ <header id="main">
+  <div id="title">
+   <h1>Dr. Norman Borlaug</h1>
+  </div>
+
+ </header>
+ <div id="img-div">
```

Comment:
Thank you for this, it is always good to have a different container for each section (or components) of our page :+1: 

Having said that, any container (div, section, article, main, etc) can be used to separate sections of the page. Therefore, "separation of header from the main body" neither enough nor genuine reason to require for this change. 

It would have been better said as: 
`Why don't you use the semantic tag "<main>" here instead of a "<div>" tag, that would improve the HTML structure and make the page more accessible`